File: AFBack/DTOs/BoostrapDTO/Critical/CriticalBootstrapResponseDTO.cs


namespace AFBack.DTOs.BoostrapDTO;

public class CriticalBootstrapResponseDTO
{
    public UserSummaryDto User { get; set; } = null!;
    public UserSettingsDTO Settings { get; set; } = null!;
    public string SyncToken { get; set; } = string.Empty;
}

File: shared/types/bootstrap/CriticalBootstrapResponseDTO.ts
import { UserBootstrapDTO } from "../UserBootstrapDTO";
import { UserProfileDTO } from "../UserProfileDTO";
import { UserSettingsDTO } from "../UserSettingsDTO";
import { BlockedUserDTO } from "../BlockedUserDTO";

export interface CriticalBootstrapResponseDTO {
  user: UserBootstrapDTO;
  profile: UserProfileDTO;
  settings: UserSettingsDTO;
  blockedUsers: BlockedUserDTO[];
}
